### PR TITLE
add ipmitool: Command-line interface to IPMI-enabled devices

### PR DESCRIPTION
**Pull request by lynxis**
Maintainer: me
Compile tested: ar71xx
Run tested: ar71xx

Command-line interface to IPMI-enabled devices

### CODE REVIEW

File: admin/ipmitool/Makefile [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ipmitool
+PKG_VERSION:=1.8.18
+PKG_RELEASE:=3
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
```
Comment (diizzyy) [inline admin/ipmitool/Makefile]: Use bz2 instead, we want to save space and bandwidth.

File: admin/ipmitool/Makefile [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ipmitool
+PKG_VERSION:=1.8.18
+PKG_RELEASE:=3
```
Comment (diizzyy) [inline admin/ipmitool/Makefile]: Initially packages starts with 1 ;-)

File: admin/ipmitool/Makefile [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=@SF/$(PKG_NAME)
+PKG_MD5SUM:=55304c6c5b994784222d3944d086fa30
```
Comment (diizzyy) [inline admin/ipmitool/Makefile]: Use PKG_HASH

**Comment by lynxis** [comment]
done.

**Comment by lynxis** [comment]
*push*

**Pull request merged**